src/app/store/technical-records/technical-record-service.reducer.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { mockVehicleTecnicalRecord, mockVehicleTecnicalRecordList } from '../../../mocks/mock-vehicle-technical-record.mock';
+import { getByVIN, getByVINSuccess, getByVINFailure } from './technical-record-service.actions';
+import { initialState, TechnicalRecordServiceState, vehicleTechRecordReducer } from './technical-record-service.reducer';
+
+describe('Vehicle Technical Record Reducer', () => {
+  describe('unknown action', () => {
+    it('should return the default state', () => {
+      const action = {
+        type: 'Unknown'
+      };
+      const state = vehicleTechRecordReducer(initialState, action);
+
+      expect(state).toBe(initialState);
+    });
+  });
+
+  describe('getByVINSuccess', () => {
+    it('should set all vehicle technical records', () => {
+      const records = mockVehicleTecnicalRecordList(5);
+      const newState: TechnicalRecordServiceState = {
+        ...initialState,
+        vehicleTechRecords: records
+      };
+      const action = getByVINSuccess({ vehicleTechRecords: [...records] });
+      const state = vehicleTechRecordReducer(initialState, action);
+
+      expect(state).toEqual(newState);
+      expect(state).not.toBe(newState);
+    });
+  });
+
+  describe('getByVINSuccess', () => {
```

Comment:
can these two tests go within the same describe block? or has the describe block been copied and the description needs updating?